### PR TITLE
Add code intelligence email address for several jvm projects

### PR DESCRIPTION
**Pull request by henryrneh**
Add code intelligence email address for several jvm projects

### CODE REVIEW

File: projects/bc-java-wycheproof/project.yaml [hunk 1]
```diff
@@ -8,3 +8,5 @@ sanitizers:
   - "none"
 file_github_issue: false
 disabled: true
+vendor_ccs:
+- hlin@code-intelligence.com
```
Comment (jonathanmetzman) [inline projects/bc-java-wycheproof/project.yaml:12]: This has nothing to do with code intelligence, please remove this.

**Comment by jonathanmetzman** [comment]
> Add code intelligence email address for several jvm projects

Can you go over these and make sure these are code-intelligence integrated projects instead of doing this for every Java project?
Thanks

File: projects/bc-java-wycheproof/project.yaml (hunk 1 above)
Comment (henryrneh) [reply projects/bc-java-wycheproof/project.yaml]: I deleted 12 projects which are not onboarded by CI, sorry for the mistake!

Comment (jonathanmetzman) [reply projects/bc-java-wycheproof/project.yaml]: No worries. My comment came across harsher than I intended.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**